syft/frameworks/torch/mpc/falcon/falcon.py

```diff
@@ -0,0 +1,13 @@
+import syft
+from .faclon_helper import FalconHelper
```

Comment:
NITPICK: We can remove the ```.```

Reply:
I think it won't work, you can either import an installed module or specify the directory